src/main/kotlin/day04/Wizard.kt

```
@@ -1,13 +1,60 @@
+    _wand: Wand?,
 ) {
+    init {
+
+    }
+    companion object {
+        const val MAX_HP = 50
+        const val MAX_MP = 30
+        const val MP_FOR_HEAL = 5
+    }
+
+    var name: String? = if(_name == null || _name.length < 3) throw IllegalArgumentException() else _name
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)